distribution/ddf-common/src/main/resources/bin/setenv-windows-wrapper.conf

```diff
@@ -23,13 +23,15 @@ wrapper.java.additional.17 = -XX:+UnlockDiagnosticVMOptions
 wrapper.java.additional.18 = -XX:+DisableAttachMechanism
 wrapper.java.additional.19 = -XX:+UnsyncloadClass
 wrapper.java.additional.20 = -Djava.awt.headless=true
+wrapper.java.additional.21 = -Dddf.home=%KARAF_HOME%
+wrapper.java.additional.22 = -Dddf.home.perm=%KARAF_HOME%/
```

Comment:
This forward slash isn't going to work on Windows.

Reply:
Fixed. Changed the other paths in this file to use backslashes as well